pkg/gofr/gofr.go

```diff
@@ -235,6 +235,13 @@ func (a *App) httpServerSetup() {
 		a.add(http.MethodGet, "/.well-known/{name}", SwaggerUIHandler)
 	}
 
+	requestTimeout := a.Config.Get("REQUEST_TIMEOUT")
+	timeoutVal, err := strconv.Atoi(requestTimeout)
+
+	if (err != nil && requestTimeout != "") || timeoutVal < 0 {
+		a.container.Error("invalid value of config REQUEST_TIMEOUT.")
+	}
```

Comment:
I would like to suggest this.

```suggestion
	requestTimeout := a.Config.Get("REQUEST_TIMEOUT")
	if requestTimeout != "" {
		timeoutVal, err := strconv.Atoi(requestTimeout)
 		if err != nil || timeoutVal < 0 {
			a.container.Error("invalid value of config REQUEST_TIMEOUT.")
		}
	}
```

I think it's clearer

Reply:
Sure will refactor it